test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -32,68 +32,60 @@ Describe "Format-Table" -Tags "CI" {
 Describe "Format-Table DRT Unit Tests" -Tags "CI" {
 		It "Format-Table with not existing table with force should throw PipelineStoppedException"{
 				$obj = New-Object -typename PSObject
-				try
-				{
-						$obj | Format-Table -view bar -force -EA Stop
-						Throw "Execution OK"
-				}
-				catch
-				{
-						$_.CategoryInfo | Should Match "PipelineStoppedException"
-						$_.FullyQualifiedErrorId | Should be "FormatViewNotFound,Microsoft.PowerShell.Commands.FormatTableCommand"
-				}
+				$e = { $obj | Format-Table -view bar -force -EA Stop } | ShouldBeErrorId "FormatViewNotFound,Microsoft.PowerShell.Commands.FormatTableCommand"
```

Comment:
Since you're updating this code, you should not use `ShouldBeErrorId` here, instead use `Should -Throw -ErrorId "..." -PassThru`